src/Tizen.NUI/src/public/BaseComponents/ViewEnum.cs

```diff
@@ -206,6 +206,7 @@ internal class Property
             internal static readonly int MARGIN = Interop.ViewProperty.View_Property_MARGIN_get();
             internal static readonly int PADDING = Interop.ViewProperty.View_Property_PADDING_get();
             internal static readonly int SHADOW = Interop.ViewProperty.View_Property_SHADOW_get();
+            internal static readonly int CAPTURE_ALL_TOUCH_AFTER_START = Interop.ActorProperty.Actor_Property_CAPTURE_ALL_TOUCH_AFTER_START_get();
```

Comment:
I think it would be better to change as Pascal case.
https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/